task-manager-backend/src/task/controllers/create-task/create-task.js

```diff
@@ -0,0 +1,46 @@
+const ApiResponse = require("../../../utils/api-response");
+const ApiError = require("../../../utils/api-error");
+const { createTaskUsecase } = require("../../usecases");
+const { sendMailUsecase } = require("../../../mail/usecases");
+
+const { TASK_CREATE_SUCCESS } = require("../../../commons/constants");
+
+async function sendTaskAssignEmail(userId, adminId) {
```

Comment:
this should not be required as it only calls execute.  Call it directly in the controller method

Reply:
As sending email is not part of create-task so I created different function for it.